pyglet/window/mouse.py

```diff
@@ -64,6 +71,9 @@ def on_mouse_drag(self, x: int, y: int, dx: int, dy: int, buttons: int, modifier
 
     def __getitem__(self, key: str) -> int | bool:
         return self.data.get(key, False)
+    
+    def __getattr__(self, item: str) -> int:
+        return self.data.get(item, default=0)
```

Comment:
It may be better to avoid .get, and just use:
```py
    def __getattr__(self, item: str) -> int:
        return self.data[item]
```
Otherwise it will return '0' when querying _any_ unrecognized attribute, like `mouse.q`.